doc/classes/Rect2.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Rect2" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		2D axis-aligned bounding box using floating point coordinates.
+		2D axis-aligned bounding box, using floating-point coordinates.
```

Comment:
The comma here isn't gramatically correct, since the bounding box is the one who's using floating-point coordinates. Some ideas:

"2D axis-aligned bounding box with floating-point coordinates.

"2D axis-aligned bounding box that uses floating-point coordinates."